modules/openapi-generator/src/main/resources/dart/libraries/dio/serialization/built_value/api_util.mustache

```diff
@@ -4,8 +4,8 @@ import 'dart:typed_data';
 
 import 'package:built_collection/built_collection.dart';
 import 'package:built_value/serializer.dart';
-import 'package:dio/dio.dart';
-import 'package:dio/src/parameter.dart';
+import '{{dioImport}}';
+import 'package:{{dioLibrary}}/src/parameter.dart';
```

Comment:
won't this be included automatically if we import dio ?

Reply:
No unfortunately it's not exported from dio.dart.

Reply:
is it something we need? If it's not exposed it's probably shouldn't be used.

Reply:
It is something I actually added to dio for the 4.0 release and I forgot to export that file :(

Reply:
ha ha ok :) never mind then ^^ it's just I've check where it's used and kind of didn't understand the point of that :)

Reply:
But that is something that can be fixed in `dio_http`.
I created https://github.com/flutterchina/dio/pull/1177 for this a while back.

Reply:
maybe we can add a flag like `useDioHttp` which will be `useOriginalDio` and check it before importing `import 'package:{{dioLibrary}}/src/parameter.dart';`

Reply:
I think we currently need this import for both options.

Reply:
if we can get it exported in the dio_http package we can add some conditional logic to remove the import but for now it needs to be there.